packages/backend/src/modules/user/user.model.ts

```diff
@@ -1,28 +1,32 @@
 import mongoose from 'mongoose'
-import { Role } from './role'
+import { UserBase } from 'packages/shared/lib'
+// import { Role } from './role'
 
-const userSchema = new mongoose.Schema({
-  username: String,
-  googleId: String,
-  avatarUrl: String,
-  email: String,
-  dashboards: [
-    {
-      imageCoverId: String,
-      title: String,
-      role: {
-        type: String,
-        enum: Role,
-        default: Role.NO_ACCESS
-      },
-      users: [
-        {
-          avatarId: Number
-        }
-      ]
-    }
-  ]
+export interface UserDocument extends UserBase, mongoose.Document {}
+
+export interface UserModel extends mongoose.Model<UserDocument> {}
+
+const userSchema: mongoose.Schema<UserDocument> = new mongoose.Schema<UserDocument, UserModel>({
```

Comment:
do not change schema!

Reply:
Well it's either that or I have to implement the `dashboards` property everywhere else, which I think we agreed isn't necessary for now

I also added `required: true` to make mongoose throw an error in case some property is missing

Reply:
add missing fields to UserDocument, it can look like here:
```
export interface TaskDocument extends TaskBase, mongoose.Document {
  attachments: any[]
}
```
later we will replae it with: AttachmentDocument

Reply:
add missing fields to UserDocument, it can look like here:
```
export interface TaskDocument extends TaskBase, mongoose.Document {
  attachments: any[]
}
```
later we will replace it with: AttachmentDocument

Reply:
ok